README.md

```diff
@@ -43,10 +43,18 @@ $ npm install --save mapbox
 
 ## Usage
 
-Setup:
-
+Setup via `node`:
 ```js
```

Comment:
You probably want to move the "Installation" section with `npm install --save mapbox` to here.

Reply:
I've added a commit moving the script tag usage up into installation. But open to any suggestions to make it clear and simple for both the browser and node.